test/graph/test_graph.py

```diff
@@ -269,7 +269,7 @@ def test_lattice_symmetry(i, name):
     assert sgb.little_group(np.zeros(dimension[i])) == sgb.point_group_
 
     # Generate little groups and their irreps
-    assert len(sgb.little_group(kvec[i])) == little_group_size[i]
+    assert len(sgb.little_group(*kvec[i])) == little_group_size[i]
```

Comment:
```suggestion
    assert len(sgb.little_group(*kvec[i])) == little_group_size[i]
    assert len(sgb.little_group(kvec[i])) == little_group_size[i]
```
can we check both syntaxes?

Reply:
every plausible syntax (array, list, tuple, vararg) gets checked by some of the tests, I guess I can add this though